app/user.go

```diff
@@ -1192,7 +1192,7 @@ func (a *App) DeleteToken(token *model.Token) *model.AppError {
 	return nil
 }
 
-func (a *App) UpdateUserRoles(userId string, newRoles string) (*model.User, *model.AppError) {
+func (a *App) UpdateUserRoles(userId string, newRoles string, sendWebSocketEvent bool) (*model.User, *model.AppError) {
```

Comment:
@jwilander When would we not want to fire a websocket event for a role change? 0/5 about whether or not we keep it, but it feels like an unnecessary prop

Reply:
On bulk import mainly, and for tests